dap-etl/transformation/src/main/scala/org/broadinstitute/monster/dap/eols/RecentSymptomsTransformations.scala

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.monster.dap.eols
+
+import org.broadinstitute.monster.dap.common.RawRecord
+import org.broadinstitute.monster.dogaging.jadeschema.fragment.EolsRecentSymptom
+
+object RecentSymptomsTransformations {
+
+  /**
+    * Parse all eol_med_symptoms data out of a raw RedCap record,
+    * injecting them into a partially-modeled Eols record.
```

Comment:
Small: I like this comment, especially the note about the EOLS record being _partially_ modeled.